samples/nrf9160/modem_shell/overlay-non-offloading.conf

```diff
@@ -0,0 +1,108 @@
+#
```

Comment:
Can we clean up the commented out configs here?

Reply:
Maybe we can. Personally I like to left some of the possible configurations there for future tunings. 
Anyways, now cleaned up.